test/functional/test_framework/authproxy.py

```diff
@@ -191,3 +191,6 @@ def _get_response(self):
         else:
             log.debug("<-- [%.6f] %s"%(elapsed,responsedata))
         return response
+
+    def __truediv__(self, relative_uri):
+        return AuthServiceProxy("{}{}".format(self.__service_url, relative_uri), self._service_name, connection=self.__conn)
```

Comment:
In commit 'Fix AuthProxy multiwallet support'.

I think using / "path is slightly better since it's consistent with the Python path API, as @ryanofsky mentioned in IRC, but I don't care too strongly.

If you do chose to make this change, please squash with the previous commit.